department-of-veterans-affairs/veteran-facing-services-tools
# Updated site design

This updates the site design to follow the design system site header design. I would also like to updated the sidebar to use the Formation sidebar styles and get the mobile navigation working, but this PR is already too large.

Notes:
- The vast majority of the new styles are copied and pasted from the design system site.
- I've updated Formation to be the current version, which is where the lock files changes came from
- I've reorganized layouts some. I'm going to do this more, but for now this is ok
- I changed a class name that was in the component mdx files, which means a bunch of files changed for just that reason.
- I'm not sure if I've fixed the fonts issue on the main site, I will try to test that out before merging.

How to review:
1. Check out the main.scss file, make sure the organization makes sense.
2. Check out Layout.jsx and SidebarLayout.jsx to make sure those components look good
3. Check new styles to make sure the naming convention makes sense to you. It's essentially the design system BEM convention, but with `site` instead of `vads`.

![Screen Shot 2019-04-11 at 2 43 37 PM](https://user-images.githubusercontent.com/634932/55983217-9e145e80-5c68-11e9-815a-f1f14a50c550.png)
